tests/test_envs.py

```diff
@@ -163,3 +165,46 @@ def test_pickle_env(env_spec):
     data_equivalence(env.step(action), pickled_env.step(action))
     env.close()
     pickled_env.close()
+
+_test_robot_env_reset_list = ['Fetch', 'HandReach']
+
+@pytest.mark.parametrize(
+    "spec", 
+    [spec for spec in non_mujoco_py_env_specs if np.any([tar in spec.id for tar in _test_robot_env_reset_list])], 
+    ids=[spec.id for spec in non_mujoco_py_env_specs if np.any([tar in spec.id for tar in _test_robot_env_reset_list])]
+)
+def test_robot_env_reset(spec):
+    """Checks initial state of robotic environment, i.e. Fetch and Shadow Dexterous Hand Reach, 
+    whether their initial states align with the document."""
+
+    def _test_initial_states(env, seed=None):
+        diag_dict = {}
+
+        init_obs = env.reset(seed = seed)
+
+        if isinstance(init_obs[0], dict):
+            diag_dict.update(**init_obs[0])
+        elif isinstance(init_obs[0], np.ndarray):
+            diag_dict.update({'observation': init_obs[0]})
+        diag_dict.update({
+            'qpos': env.unwrapped.data.qpos,
+            'qvel': env.unwrapped.data.qvel,
+            'init_qpos': env.unwrapped.initial_qpos,
+            'init_qvel': env.unwrapped.initial_qvel
+        })
+
+        # exclude object location from environments
+        if np.any([tar in spec.id for tar in ['FetchPush', 'FetchPickAndPlace', 'FetchSlide',]]):
+            diag_dict['qpos'] = np.delete(diag_dict['qpos'], np.s_[-7:-5])
+            diag_dict['init_qpos'] = np.delete(diag_dict['init_qpos'], np.s_[-7:-5])
+
+        # testing
+        assert np.allclose(diag_dict['qpos'], diag_dict['init_qpos'])
+        assert np.allclose(diag_dict['qvel'], diag_dict['init_qvel'])
+        return diag_dict
+    
+    cur_env: gym.Env = spec.make()
+
+    _test_initial_states(cur_env, seed=24)
+    _test_initial_states(cur_env, seed=10)
+    return
```

Comment:
remove this `return`

Reply:
Is there a particular reason for this?

Reply:
No reason to have a trailing `return`, Farama codding style